src/category_theory/abelian/injective_resolution.lean

```diff
@@ -180,4 +180,101 @@ end abelian
 
 end InjectiveResolution
 
+section
+variables [abelian C]
```

Comment:
Where exactly are you using `exact g.op f.op`? I don't see it at first glance.

Reply:
Way above, at here
https://github.com/leanprover-community/mathlib/blob/ad383213794364e36e3d29f8a9573af6c7c3b646/src/category_theory/abelian/injective_resolution.lean#L72

Reply:
Aah, thanks for the pointer.